pkg/envoy/ingress_translator.go

```diff
@@ -147,6 +148,22 @@ func classFilter(ingresses []v1beta1.Ingress, ingressClass []string) []v1beta1.I
 	return is
 }
 
+func validIngressFilter(ingresses []v1beta1.Ingress) []v1beta1.Ingress {
+	vi := make([]v1beta1.Ingress, 0)
+
+	for _, i := range ingresses {
+		for _, j := range i.Status.LoadBalancer.Ingress {
+			if j.Hostname == "" && j.IP == "" {
+				logrus.Debugf("no hostname or ip for loadbalancer found in ingress config")
```

Comment:
I think it would be useful for debugging in this logged the name/namespace of the ingress here